plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -355,7 +362,8 @@ public void testLimitPushdown()
         assertThat(query("SELECT name FROM nation WHERE regionkey = 3 LIMIT 5")).isFullyPushedDown();
 
         // with filter over varchar column
-        assertThat(query("SELECT name FROM nation WHERE name < 'EEE' LIMIT 5")).isFullyPushedDown();
+        assertThat(query("SELECT name FROM nation WHERE name < 'EEE' LIMIT 5"))
+                .isNotFullyPushedDown(FilterNode.class);
```

Comment:
add comment here too please.

Reply:
Hmm - maybe we `isNotFullyPushedDown` should take `String reason` parameter so we enforce that?
@findepi ?